besu/src/main/java/org/hyperledger/besu/cli/options/unstable/DataStorageOptions.java

```diff
@@ -29,10 +29,10 @@
 
 public class DataStorageOptions implements CLIOptions<DataStorageConfiguration> {
 
-  private static final String DATA_STORAGE_FORMAT = "--Xdata-storage-format";
+  private static final String DATA_STORAGE_FORMAT = "--data-storage-format";
```

Comment:
as per comments, can the package name here be changed to remove "unstable"

Reply:
[8bd3d10](https://github.com/hyperledger/besu/pull/3578/commits/8bd3d10e473dc7fc2984c924161b484ebadd1e48)